StRoot/TPCCATracker/StTPCCAInterface.cxx

```diff
@@ -0,0 +1,397 @@
+//#define __ESTIMATE_Primary_Vertex_Z__
+#include "StTPCCAInterface.h"
+#include "TPCCATracker/AliHLTTPCCAGBHit.h"
+#include "TPCCATracker/AliHLTTPCCAGBTrack.h"
+#include "TPCCATracker/AliHLTTPCCAParam.h"
+// need for hits data
+#include "StMaker.h"
+#include "StTpcHit.h"                
+#include "StTpcDb/StTpcDb.h"
+#include "StDbUtilities/StTpcCoordinateTransform.hh"
+#include "StDbUtilities/StTpcLocalSectorCoordinate.hh"
+  // for MCdata
+#include "tables/St_g2t_track_Table.h" 
+#include "tables/St_g2t_tpc_hit_Table.h"
+#include "TDatabasePDG.h"
```

Comment:
Not used?

Reply:
Can we keep them for now?

Reply:
No 😄  Unused includes are not free, they do slow down the build. Ideally, someone should run a utility to identify unused headers.